src/KanbanModal.js

```diff
@@ -34,22 +34,24 @@ const KanbanModal = (props) => {
             props.addTask(newTask);
           }}
         >
-          <div className="KanbanModal-input-container">
+          <div style={{display: 'flex'}}>
             <TextField
               id="filled-search"
               label="Task:"
               type="search"
               value={task}
+              variant="standard"
               onChange={handleChangeTask}
+              style={{padding: 6}}
             />
-          </div>
-          <div className="KanbanModal-input-container">
             <TextField
               id="filled-search"
               label="For:"
               type="search"
               value={user}
+              variant="standard"
               onChange={handleChangeUser}
+              style={{padding: 6}}
```

Comment:
Same comment on CSS file. If we're making any further change this would just further complicate things.

Reply:
this is because I used a material ui component to replace whatever ugly class that was there

Reply:
Unless you're relocating all the original CSS, having things in 2 different places just makes it more difficult to manage

Reply:
We're not exactly aiming for clean code here. Some of the material ui components are already pretty enough but some just need one tiny css tweak, if you insist I can move these CSS one-liners to those .css files